### ISSUE TITLE
Micro-site: categorize and highlight presentations w/ better index

### ISSUE THREAD

**Issue by ultrasaurus** [cncf/tag-security#110]
*Description*: we want to surface the work that the group has done over the past 1.5 years and create a maintainable structure, so as we add more to the repo, parts of it will naturally update on the web also -- basically, more accessible content with some friendly pointers to the repo. We want to start small and iterate, so we thought starting with the presentation would be useful to people and relatively easy to put together.  Other content (home page about) could be taken from readme, potentially refactoring parts into separate docs, if needed, to not have content replicated in multiple places.  This isn't a site about the SIG, it's a site about cloud native security (knowledge sharing by SIG-Security).

*Impact*: Make the work of the group more accessible to a larger audience.  Initial target audience is people who are already fairly knowledgable about cloud or about security (e.g. new group members), later expanding as the group creates more resources.  

*Scope*: Initial version should take a few hours to 1 day of work to make the site...  once we have all ther prerequisites figure out and a plan with review checkpoints, it could get done 3-4 weeks of calendar time for iterations and discussion to figure out exactly how to set up the files so that they are both easy to maintain and readable, allowing for at least a week in the middle for review/feedback from the wider group.

We have a lot of great source material about security use cases as well as from presentations from specific open source projects that provide solutions in this space

If you are interested in getting involved, pls comment on this issue and join #sig-security-web channel on slack

proposed directory structure:

/presentations
* /use-case
* /security-provider

I've gotten a transcript for each session that I plan to post and with overview page for each, including a link to github issue, video, transcript, etc.

- Project lead: 
- SIG Chair: @ultrasaurus 

TODO:
- [ ] post raw info from presentations: https://github.com/cn-security/safe/issues?q=label%3Ausecase-presentation+is%3Aclosed (@taylorwaggoner), [tracking sheet](https://docs.google.com/spreadsheets/d/1eTibQmR6g2CJdgoIKH_3ylXtpM73KY32po8T-bL-SWg/edit#gid=0)
   - [X] videos on YouTube - [playlist](https://www.youtube.com/watch?v=xuimT5lN7t8&list=PLj6h78yzYM2NRxVK1VUal0fCRi4XGm0xV)
   - [ ] get transcripts 
- [ ] figure out format for the pages (see notes below)
- [ ] make a markdown page for each presentation with an index 
- [ ] pick a hugo template
- [ ] find out from CNCF 
  - [ ] are any color / visual style guidelines?
  - [ ] at what domain will this live?
  - [ ] approval process?  (plan to share with SIG, approval by majority of co-chairs, and Joe/Liz -- though need to determine some checkpoints so a big group isn't approving every PR)

Notes:
-  idea for /transcripts folder https://github.com/cn-security/safe/pull/114 
-  ideas on structured meta data: https://github.com/ultrasaurus/safe/tree/presentation-page-format/presentations

**Comment by ultrasaurus** [comment]
I'm thinking we should consider a static site generator like [Hugo](https://gohugo.io) -- then we can keep meta-data in machine readable front-matter (though we could publish markdown as interim step, since I read that github now displays yaml front-matter)

**Comment by ultrasaurus** [comment]
here are some ideas of data representations for the presentations: https://github.com/ultrasaurus/safe/tree/presentation-page-format/presentations

**Comment by izgeri** [comment]
I like the idea of having a directory structure that is indicative of the type of meeting transcripts that are included in that dir. Maybe each dir could have its own README with an index of the file inside? Or we could use a static site generator.

**Comment by mhausenblas** [comment]
@ultrasaurus count me in. I've done a couple of sites with GitHub/Hugo combo already, the main question is where to host it. I usually use Amplify but I acknowledged the fact that Netlify is more popular (and arguably I'm biased re the former ;)

**Comment by petermbenjamin** [comment]
Have we determined if this micro-site going to live as a subdomain on cncf.io or on its own domain?

**Comment by ultrasaurus** [comment]
@petermbenjamin added to TODO list -- need to follow-up with Amye to get an intro to the CNCF person who can answer whether there are any preferences on their side for URLs or visual styling.

I did nab `cloud-native-security.info` a while ago (before we started the CNCF WG / SIG process) and happy to donate that if people like it and we want / need our own domain.  Though I do like `security.cncf.io` which seems more concise and readable.

**Comment by mhausenblas** [comment]
@ultrasaurus would you mind stating if you take care of it or want me to do it?

**Comment by ultrasaurus** [comment]
@mhausenblas will kick of an email thread to find the right person to coordinate with and cc you.  It would be great if you could track that thread and update here as we learn answers

**Comment by caniszczyk** [comment]
hey @lucperkins from the CNCF can sketch out a simple hugo+netlify site

we can do something like sig-security.cncf.io

**Comment by mhausenblas** [comment]
Thanks, @ultrasaurus … yo @lucperkins can we sync regarding the micro-site, this week, please? My current understanding is that you'd be looking after infra (Hugo, hosting, etc.) and I more after the content? I suppose a quick (20min) meeting to resolve it would be ideal?

**Comment by lucperkins** [comment]
Status update: I'm working on something that's publicly available at https://sig-security.netlify.com. I'm building that from my personal fork of this repo: https://github.com/lucperkins/sig-security/tree/lperkins/website. It's definitely a WIP but I think a decent skeleton for iteration.

If it would be beneficial, I can submit a PR and we can discuss/collaborate there.

**Comment by mhausenblas** [comment]
@lucperkins: today @ultrasaurus and I sat together and developed a battle plan. Here's what we decided:

- We continue to use your fork, please use #195 as the landing page content
- Please add a client-side site search functionality
- I will work on the presentation details content in my own fork, based on Sarah's [templates](https://github.com/ultrasaurus/cncf-sig-security/blob/presentation-page-format/presentations/test-links.md)

We can discuss details or open questions on the `sig-security-web` channel.

**Comment by lucperkins** [comment]
@mhausenblas I've updated the site in progress with the desired front page content as well as a search bar: https://sig-security.netlify.com/.

I've left the previous documents in place largely to demonstrate the search functionality. They'll be removed later.

**Comment by mhausenblas** [comment]
Awesome, thank you @lucperkins!

**Comment by mhausenblas** [comment]
Sorry for the long silence. Week after next (re:Invent) I'm back on track and wanted to see where we are with this issue and where/how I can contribute @ultrasaurus

**Comment by chasemp** [comment]
interested :)

**Comment by TheFoxAtWork** [comment]
@vinayvenkat tagging you here for awareness/interest

we'll need to define an editorial team to curate content, field content.  this applies to the microblog #451  issue as well - at least to get started.

@pragashj mentioned  a potential subdomain upcoming?  security.cncf.io ?  need POC to confirm and help figure this out.

**Comment by apmarshall** [comment]
Happy to help with this.

**Comment by chasemp** [comment]
Yep, seems like a great idea for discoverability and lots of stated points.

**Comment by lumjjb** [comment]
I would like to see if we can create some alignment with the work around #551 , I think there could be possibly be a part of the same site as the CNSmap, https://cnsmap.vercel.app/

So maybe we could have Cloud Native Security Map be one section and have "Presentations" or other subsections in which everyone can contribute to. The current templating we have for the CNSmap is based on markdown, so technically we could work it in a way to reflect content of the repo well.

**Comment by lumjjb** [comment]
Updating the issue comment for freshness of issue state

**Comment by lumjjb** [comment]
The plan is to have the CNSMap site evolve to become the microsite. The next usecase to expand on would be the Cloud native security lexicon #735 .

**Comment by PushkarJ** [comment]
K8s SIGs have something similar as a service for sub-projects: https://github.com/kubernetes/community/blob/master/github-management/subproject-site-requests.md

**Comment by IAXES** [comment]
Following up on a discussion w/ @lumjjb on this ticket: I've helped a few friends + colleagues use ReadTheDocs/sphinx/`rST` for sites. Works quite nicely for developer-driven documents, easy to setup a build + publish pipeline in Concourse and/or GitHub Actions. Also supports citations, cross-referencing, "code as documentation", etc.

**Examples:**
- https://iaxes.github.io/pronk8s/sections/build_patterns.html
- https://fd.io/docs/vpp/v2101/reference/readthedocs/index.html

I'd be able to throw together a demo pretty quickly, if there's interest.

**Comment by ultrasaurus** [comment]
There were plenty of people willing to do the technical work, include CNCF staff who are available to help.  What this project has always lacked is someone who is willing to curate content and, as needed, write/edit overview text for sections of the site.

**Comment by vicenteherrera** [comment]
I'll help you, will build a first alpha prototype that we can use to discuss the information, content and appearance. ;-)

**Comment by ragashreeshekar** [comment]
@ultrasaurus I can help on the curation.
Here is a first draft of the content that I think we can cover 
`
# Who we are

-   Our Charter - [https://github.com/cncf/tag-security/blob/main/governance/charter.md#sig-security-charter](https://github.com/cncf/tag-security/blob/main/governance/charter.md#sig-security-charter)
-   History
	-   TAG-Security - renamed STAG ([TOC Issue 549](https://github.com/cncf/toc/issues/549))
	-   SAFE WG - renamed to CNCF Security TAG
	-   [(Proposed) CNCF Policy Working Group](https://github.com/cncf/tag-security/blob/main/policy-wg-merging.md)  - Merged into SAFE WG

-   Our brand - [https://github.com/cncf/tag-security/tree/main/design](https://github.com/cncf/tag-security/tree/main/design)
-   Our team
	-   Chairs
	-   Tech Leads
	-   Contributors

# What we've done

-   Repository - [https://github.com/cncf/tag-security](https://github.com/cncf/tag-security)
-   Papers
	-   Cloud Native 8 - [https://github.com/cncf/tag-security/blob/main/security-whitepaper/secure-defaults-cloud-native-8.md](https://github.com/cncf/tag-security/blob/main/security-whitepaper/secure-defaults-cloud-native-8.md)
	-   Cloud Native Security Whitepaper
		-   V2 (2022) - [https://github.com/cncf/tag-security/tree/main/security-whitepaper/v2](https://github.com/cncf/tag-security/tree/main/security-whitepaper/v2)
		-   V1 (2020) - [https://github.com/cncf/tag-security/tree/main/security-whitepaper/v1](https://github.com/cncf/tag-security/tree/main/security-whitepaper/v1)
	-   Supply chain Security Whitepaper - [https://github.com/cncf/tag-security/tree/main/supply-chain-security/supply-chain-security-paper](https://github.com/cncf/tag-security/tree/main/supply-chain-security/supply-chain-security-paper)
	-   Secure Software Factory Whitepaper - [https://github.com/cncf/tag-security/tree/main/supply-chain-security/secure-software-factory](https://github.com/cncf/tag-security/tree/main/supply-chain-security/secure-software-factory)
	-   Cloud Native Security Controls Catalog - [https://github.com/cncf/tag-security/blob/main/cloud-native-controls/phase-one-announcement.md](https://github.com/cncf/tag-security/blob/main/cloud-native-controls/phase-one-announcement.md)
	-   Cloud Native Security Map - [https://github.com/cncf/tag-security/tree/main/security-whitepaper/cnsmap](https://github.com/cncf/tag-security/tree/main/security-whitepaper/cnsmap)
	-   Cloud Native Security Lexicon - [https://github.com/cncf/tag-security/tree/main/security-lexicon](https://github.com/cncf/tag-security/tree/main/security-lexicon)

-   Presentations
-   Security assessments

	-   Buildpacks - [https://github.com/cncf/tag-security/tree/main/assessments/projects/buildpacks](https://github.com/cncf/tag-security/tree/main/assessments/projects/buildpacks)
	-   Cloud Custodian - [https://github.com/cncf/tag-security/tree/main/assessments/projects/custodian](https://github.com/cncf/tag-security/tree/main/assessments/projects/custodian)
	-   Harbor - [https://github.com/cncf/tag-security/tree/main/assessments/projects/harbor](https://github.com/cncf/tag-security/tree/main/assessments/projects/harbor)
	-   In-toto - [https://github.com/cncf/tag-security/tree/main/assessments/projects/in-toto](https://github.com/cncf/tag-security/tree/main/assessments/projects/in-toto)
	-   Keycloak - [https://github.com/cncf/tag-security/tree/main/assessments/projects/keycloak](https://github.com/cncf/tag-security/tree/main/assessments/projects/keycloak)
	-   Kyverno - [https://github.com/cncf/tag-security/tree/main/assessments/projects/kyverno](https://github.com/cncf/tag-security/tree/main/assessments/projects/kyverno)
	-   OPA - [https://github.com/cncf/tag-security/tree/main/assessments/projects/opa](https://github.com/cncf/tag-security/tree/main/assessments/projects/opa)
	-   Spiffe-Spire - [https://github.com/cncf/tag-security/tree/main/assessments/projects/spiffe-spire](https://github.com/cncf/tag-security/tree/main/assessments/projects/spiffe-spire)

-   Policy - [https://github.com/cncf/tag-security/tree/main/policy](https://github.com/cncf/tag-security/tree/main/policy)
-   Events - [https://github.com/cncf/tag-security/blob/main/cloud_native_security.md](https://github.com/cncf/tag-security/blob/main/cloud_native_security.md)

# How do we do this

[STAG charter](https://github.com/cncf/tag-security/blob/main/governance/charter.md)  outlines the scope of our group activities, as part of our  [governance process](https://github.com/cncf/tag-security/blob/main/governance)  which details how we work.

# Top 5 things we are working on

-   SecurityCon at KubeCon+CloudNativeCon NA 2022 - [https://github.com/cncf/tag-security/issues/939](https://github.com/cncf/tag-security/issues/939)
-   Cloud Native Security Controls Catalog v2 - [https://github.com/cncf/tag-security/issues/845](https://github.com/cncf/tag-security/issues/845)
-   Serverless Security Whitepaper - [https://github.com/cncf/tag-security/issues/546](https://github.com/cncf/tag-security/issues/546)
-   Cloud Native Security Map v2 - [https://github.com/cncf/tag-security/issues/737](https://github.com/cncf/tag-security/issues/737)
-   Cloud Native Security Whitepaper - Audio project - [https://github.com/cncf/tag-security/issues/606](https://github.com/cncf/tag-security/issues/606)

# How to get involved
-   Contributors
	-   Slack channel
		-   Hangout in the CNCF Slack
		-   Our channel is #tag-security

	-   Mailing list
		-   Join our mailing list at CNCF Lists TAG-Security
		-   Join the Mailing List to receive the calendar meeting invite.

	-   Meeting calendar
		-   CNCF Event Calendar
		-   We meet every Wednesday at 10 am PT| 1 pm ET| 6 pm GMT

	-   Zoom
		-   Join our call live on Zoom
		-   cncftagsecurity passcode: 77777

	-   YouTube channel
		-   Missed a meeting?
		-   No drama - catch up from a recording!

	-   Contact us
		-   Chairs - cncf-tag-security-chairs@lists.cncf.io
		-   Chairs & Tech Leads - cncf-tag-security-chairs@lists.cncf.io

-   Security assessments
	- Guide - [https://github.com/cncf/tag-security/tree/main/assessments/guide](https://github.com/cncf/tag-security/tree/main/assessments/guide)
	- Contact us
		- Chairs - cncf-tag-security-chairs@lists.cncf.io
		- Chairs & Tech Leads - cncf-tag-security-chairs@lists.cncf.io`

Related issues
- https://github.com/cncf/tag-security/issues/666
- https://github.com/cncf/tag-security/issues/826

Potentially related issues
- https://github.com/cncf/tag-security/issues/692
- https://github.com/cncf/tag-security/issues/726

**Comment by vicenteherrera** [comment]
Thanks for the proposed structure, I'll try to incorporate that to my test!

I've been preparing a test website, you can see it at: https://vicenteherrera.com/stag-web/
Code lives at the repo here: https://github.com/vicenteherrera/stag-web

Some interesting characteristics about it for this group:
* Based on Jekyll (an open source ruby tool) and an [open source theme](https://github.com/kitian616/jekyll-TeXt-theme)
* It generates a static website (plain HTML), that can be hosted on GitHub pages (as it shows in the [test repo here](https://github.com/vicenteherrera/stag-web)), so less vulnerabilities to deal with, very quick rendering, and using GitHub as a free host (you can even associate your own domain to GitHub pages as I did).
* Editing content is easy even if you don't know about web development using just [markdown files like this one](https://raw.githubusercontent.com/vicenteherrera/stag-web/master/about.md) (that generates [this url](https://vicenteherrera.com/stag-web/about.html)) Even blog posts are just markdown files.
* When somebody wants to propose a change, he/she can open a PR that can be commented and accepted. You can render and browse a local version on your computer to check modifications.
* The theme is quite clean and nice, it includes possibility of [two menu navigation levels](https://tianqi.name/jekyll-TeXt-theme/page/sidebar-aside.html) (horizontal and vertical) that gives flexibility to incorporate a lot of static pages not associated specifically to blog posts, as well as [many other options](https://tianqi.name/jekyll-TeXt-theme/samples.html).

I plan to work putting @ragashreeshekar shared content into this test, and when it's more mature, we can discuss the best way to really publish it and manage it.

**Comment by ultrasaurus** [comment]
this is great to see!   Thought I would chime in with some historical notes, in case it's helpful..

When we were looking at this long ago, we were thinking of having it be a view of the main repo (with slight changes to structure if needed), so that new content would also appear in the repo itself.  It has been a long time since I used Jekyll, but I think most static site generators have flexibility to pull templates from a sub-directory and markdown content from multiple directories.  Just a thought.  Happy to talk sometime 1-1 or small group if helpful (just reach out on Slack)

**Comment by ragashreeshekar** [comment]
@ultrasaurus Thanks, @vicenteherrera, and I will surely get connected to you.
Hope the initial approach is ok, and we will reach out with some progress

**Comment by vicenteherrera** [comment]
@ultrasaurus You are right. There are several options for a GitHub page: root folder, specific folder, different branch, and of course, different repository.
I believe same repository and using a specific directory (like "website") may be the better approach. That way you have the website content version linked to the rest of the content of the repository very easily, and the same review process to accept a PR should be in place.
In my case, right now I just want to build a proof of concept, where I can experiment without tainting the original repo with branches, and a lot of false start commits. Right now I've tested already three different themes (from which I forked their own repositories), and I think I will transition from having all the files of the theme in the repo, to just reference it for a cleaner directory, so people that doesn't know Jekyll a lot don't have to filter those out to get to the relevant ones.
But I think this theme is useful for us, and with @ragashreeshekar proposed structure it won't take long until we have a POC that we can use to discuss the final content and move to the main repository.

**Comment by ragashreeshekar** [comment]
> @ultrasaurus I can help on the curation. Here is a first draft of the content that I think we can cover @vicenteherrera that's some good work already, thanks. Please let me know your thoughts as well, and how we could articulate this in the site you are building. `
> 
> # Who we are
> * Our Charter - https://github.com/cncf/tag-security/blob/main/governance/charter.md#sig-security-charter
> * History
>   
>   * TAG-Security - renamed STAG ([TOC Issue 549](https://github.com/cncf/toc/issues/549))
>   * SAFE WG - renamed to CNCF Security TAG
>   * [(Proposed) CNCF Policy Working Group](https://github.com/cncf/tag-security/blob/main/policy-wg-merging.md)  - Merged into SAFE WG
> * Our brand - https://github.com/cncf/tag-security/tree/main/design
> * Our team
>   
>   * Chairs
>   * Tech Leads
>   * Contributors
> 
> # What we've done
> * Repository - https://github.com/cncf/tag-security
> * Papers
>   
>   * Cloud Native 8 - https://github.com/cncf/tag-security/blob/main/security-whitepaper/secure-defaults-cloud-native-8.md
>   * Cloud Native Security Whitepaper
>     
>     * V2 (2022) - https://github.com/cncf/tag-security/tree/main/security-whitepaper/v2
>     * V1 (2020) - https://github.com/cncf/tag-security/tree/main/security-whitepaper/v1
>   * Supply chain Security Whitepaper - https://github.com/cncf/tag-security/tree/main/supply-chain-security/supply-chain-security-paper
>   * Secure Software Factory Whitepaper - https://github.com/cncf/tag-security/tree/main/supply-chain-security/secure-software-factory
>   * Cloud Native Security Controls Catalog - https://github.com/cncf/tag-security/blob/main/cloud-native-controls/phase-one-announcement.md
>   * Cloud Native Security Map - https://github.com/cncf/tag-security/tree/main/security-whitepaper/cnsmap
>   * Cloud Native Security Lexicon - https://github.com/cncf/tag-security/tree/main/security-lexicon
> * Presentations
> * Security assessments
>   
>   * Buildpacks - https://github.com/cncf/tag-security/tree/main/assessments/projects/buildpacks
>   * Cloud Custodian - https://github.com/cncf/tag-security/tree/main/assessments/projects/custodian
>   * Harbor - https://github.com/cncf/tag-security/tree/main/assessments/projects/harbor
>   * In-toto - https://github.com/cncf/tag-security/tree/main/assessments/projects/in-toto
>   * Keycloak - https://github.com/cncf/tag-security/tree/main/assessments/projects/keycloak
>   * Kyverno - https://github.com/cncf/tag-security/tree/main/assessments/projects/kyverno
>   * OPA - https://github.com/cncf/tag-security/tree/main/assessments/projects/opa
>   * Spiffe-Spire - https://github.com/cncf/tag-security/tree/main/assessments/projects/spiffe-spire
> * Policy - https://github.com/cncf/tag-security/tree/main/policy
> * Events - https://github.com/cncf/tag-security/blob/main/cloud_native_security.md
> 
> # How do we do this
> [STAG charter](https://github.com/cncf/tag-security/blob/main/governance/charter.md) outlines the scope of our group activities, as part of our [governance process](https://github.com/cncf/tag-security/blob/main/governance) which details how we work.
> 
> # Top 5 things we are working on
> * SecurityCon at KubeCon+CloudNativeCon NA 2022 - [SecurityCon at KubeCon+CloudNativeCon NA 2022  #939](https://github.com/cncf/tag-security/issues/939)
> * Cloud Native Security Controls Catalog v2 - [Cloud Native Security Controls Mapping to NIST ( Phase II for #635) #845](https://github.com/cncf/tag-security/issues/845)
> * Serverless Security Whitepaper - [Serverless Security White Paper - Cloud Native Security Best Practices  #546](https://github.com/cncf/tag-security/issues/546)
> * Cloud Native Security Map v2 - [[Proposal] CNCF Cloud Native Security Map - v2 #737](https://github.com/cncf/tag-security/issues/737)
> * Cloud Native Security Whitepaper - Audio project - [Audio recordings of CNCF Security Papers #606](https://github.com/cncf/tag-security/issues/606)
> 
> # How to get involved
> * Contributors
>   
>   * Slack channel
>     
>     * Hangout in the CNCF Slack
>     * Our channel is #tag-security
>   * Mailing list
>     
>     * Join our mailing list at CNCF Lists TAG-Security
>     * Join the Mailing List to receive the calendar meeting invite.
>   * Meeting calendar
>     
>     * CNCF Event Calendar
>     * We meet every Wednesday at 10 am PT| 1 pm ET| 6 pm GMT
>   * Zoom
>     
>     * Join our call live on Zoom
>     * cncftagsecurity passcode: 77777
>   * YouTube channel
>     
>     * Missed a meeting?
>     * No drama - catch up from a recording!
>   * Contact us
>     
>     * Chairs - [cncf-tag-security-chairs@lists.cncf.io](mailto:cncf-tag-security-chairs@lists.cncf.io)
>     * Chairs & Tech Leads - [cncf-tag-security-chairs@lists.cncf.io](mailto:cncf-tag-security-chairs@lists.cncf.io)
> * Security assessments
>   
>   * Guide - https://github.com/cncf/tag-security/tree/main/assessments/guide
>   * Contact us
>     
>     * Chairs - [cncf-tag-security-chairs@lists.cncf.io](mailto:cncf-tag-security-chairs@lists.cncf.io)
>     * Chairs & Tech Leads - [cncf-tag-security-chairs@lists.cncf.io](mailto:cncf-tag-security-chairs@lists.cncf.io)`
> 
> Related issues
> 
> * [[Suggestion] Improve new member experience, allow easier involvement/induction #666](https://github.com/cncf/tag-security/issues/666)
> * [[Proposal] "landing page" for new visitors #826](https://github.com/cncf/tag-security/issues/826)
> 
> Potentially related issues
> 
> * [Security TAG Communities (Community Representative/Manager) #692](https://github.com/cncf/tag-security/issues/692)
> * [[Suggestion] Improvements to CONTRIBUTING guide #726](https://github.com/cncf/tag-security/issues/726)

@vicenteherrera, please share if there is any update. 

Some of the planned content is now available within the TAG repository: 
1. Who we are, what do we do, and how do we do them? - https://github.com/cncf/tag-security/#readme
2. Published whitepapers, reference architecture and resources are available at https://github.com/cncf/tag-security/blob/main/PUBLICATIONS.md
3. Resources for projects - https://github.com/cncf/tag-security/tree/main/project-resources

**Comment by anvega** [comment]
Closing now that we have https://tag-security.cncf.io/. To further the discussion on refinement and maintenance of the site, feel free to file issues against the site repository directly.
